.github/workflows/pypi.yaml

```diff
@@ -9,21 +9,21 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.7
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip build
-    - name: Build the servicex wheel
-      env:
-        servicex_version: ${{ github.ref }}
-      run: |
-        python -m build --sdist --wheel
-    - name: Publish servicex to PyPI
-      uses: pypa/gh-action-pypi-publish@v1.3.1
-      with:
-        user: __token__
-        password: ${{ secrets.pypi_password_servicex }}
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip build
+      - name: Build the servicex wheel
+        env:
+          servicex_version: ${{ github.ref }}
+        run: |
+          python -m build --sdist --wheel
+      - name: Publish servicex to PyPI
+        uses: pypa/gh-action-pypi-publish@v1.6.4
+        with:
+          password: ${{ secrets.pypi_password }}
+          print_hash: true
```

Comment:
Not sure why all this indenting seems messed up now after the branch merge. :?

Reply:
To reduce commit noise here I'm just going to override with a push of my rebased local version as the extra spaces make it hard to track in the future what the motivation for this was.

Reply:
Thanks!